server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/PartitionFilesSorter.java

```diff
@@ -252,30 +262,38 @@ private void reloadSortedShuffleFiles(DB db) {
   }
 
   @VisibleForTesting
-  public Set<String> initSortedShuffleFiles(String shuffleKey) {
-    return sortedShuffleFiles.computeIfAbsent(shuffleKey, v -> ConcurrentHashMap.newKeySet());
+  public void clearSortedShuffleFilesInDB() {
+    for (String shuffleKey : sortedShuffleFiles.keySet()) {
+      try {
+        sortedFilesDb.delete(dbShuffleKey(shuffleKey));
```

Comment:
Maybe we should clear all contents in the db? It is unreasonable that db contains shuffleKeys not in sortedShuffleFiles

Reply:
How about current?